hwdb.d/60-sensor.hwdb

```diff
@@ -808,6 +808,14 @@ sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.12:bd07/17/201
 sensor:modalias:acpi:BMI0160*:dmi:*:rnONEXPLAYER:rvrV01:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, -1
 
+#########################################
+# Passion
+#########################################
+
+# Passion P612F
+sensor:modalias:acpi:MXC6655*:dmi:*:svnDefaultstring*:pnP612F:*
```

Comment:
"Defaultstring", noice, gotta love hardware manufacturers.

Reply:
Maybe, better to use `:svn*:`?

Reply:
> "Defaultstring", noice, gotta love hardware manufacturers.

This device does not have a vendor listed in the DMI.
Command output `cat /sys/class/dmi/id/modalias`:
`dmi:bvnAmericanMegatrendsInternational,LLC.:bvr5.19:bd10/21/2022:br5.19:efr0.36:svnDefaultstring:pnP612F:pvrDefaultstring:rvnEii:rnP612F:rvrDefaultstring:cvnDefaultstring:ct13:cvrDefaultstring:sku0001:`

Reply:
> Maybe, better to use `:svn*:`?

The use of "*" may be way too generic. If another vendor (with a normal name in dmi) suddenly makes a device with the same model name, it may be erroneously reconfigured.

Reply:
> > Maybe, better to use `:svn*:`?
> 
> The use of "*" may be way too generic. If another vendor (with a normal name in dmi) suddenly makes a device with the same model name, it may be erroneously reconfigured.

OK.